packages/ckeditor5-list/tests/listproperties/ui/listpropertiesview.js

```diff
@@ -567,6 +567,76 @@ describe( 'ListPropertiesView', () => {
 				view.keystrokes.press( keyEvtData );
 				sinon.assert.callCount( keyEvtData.stopPropagation, 4 );
 			} );
+
+			describe( 'special keys keydown event', () => {
+				let preventSpecialNumericValuesSpy;
+
+				beforeEach( () => {
+					preventSpecialNumericValuesSpy = sinon.spy( view, '_preventSpecialNumericValues' );
+				} );
+
+				it( 'should not prevent from typing digits in the start index field', () => {
+					const keyEvent = {
+						key: '0',
+						keyCode: 48,
+						shiftCode: false
+					};
+					view.startIndexFieldView.fieldView.element.dispatchEvent( new KeyboardEvent( 'keydown', keyEvent ) );
+					const args = preventSpecialNumericValuesSpy.args[ 0 ];
+
+					expect( args[ 0 ].isTrusted ).to.equal( false );
```

Comment:
This check is useless in our case.

Reply:
yes, it was intentionally left for reviewer with a comment of mine (above) to explain context.